distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -387,9 +388,10 @@ type Date_Time
              from Standard.Base import Date_Time
              import Standard.Base.Data.Time.Duration
 
-             example_plus = Date_Time.new 2020 + 15.years + 3.hours
-    + : Duration -> Date_Time
-    + self amount = self.plus_builtin amount
+             example_plus = Date_Time.new 2020 + 15.years + (Duration.new hours=3)
+    + : Duration -> Date_Time ! Polyglot_Error
+    + self amount =
+        Panic.catch ArithmeticException (self.plus_builtin amount) (err -> Error.throw (Polyglot_Error_Data err.getMessage))
```

Comment:
I don't think this should be wrapped in `Polyglot_Error_Data`. This is a wrapper in which Java exceptions are usually wrapped - so the field `cause` is supposed to be the exception, not a `Text` message. And we have never (IIRC) used `Polyglot_Error` as a dataflow error - it's a marker for Java exceptions coming from builtin/polyglot calls _and_ there are plans to remove it too. So I'm against using it in this context.

Please use some more fitting exception. Maybe `Illegal_State_Error`? Or maybe `Time_Error` - I think it is fitting what we need here.